app/components/kpi/component.js

```diff
@@ -0,0 +1,93 @@
+import Component from '@glimmer/component';
+import { service } from '@ember/service';
+import { format, getDaysInMonth } from 'date-fns';
+import { action } from '@ember/object';
+
+export default class KpiComponent extends Component {
+  @service store;
+  @service session;
+
+  get tableOfMonths() {
+    return Array.from({ length: 12 }, (_, index) =>
+      format(new Date(2022, index), 'MMMM')
+    );
+  }
+
+  get allClosedRequests() {
+    return this.args.model.filter(({ status }) => status === 'closed');
+  }
+
+  get requestTypes() {
+    return [...new Set(this.args.model.map(({ type }) => type))];
```

Comment:
I know - it's weird but Ember inject some extra methods to native array, for this case you can use `mapBy` https://api.emberjs.com/ember/4.8/classes/MutableArray/methods/mapBy?anchor=mapBy
```suggestion
    return [...new Set(this.args.model.mapBy('type')];
```